radicle-cli/src/commands/id.rs

```diff
@@ -199,29 +200,29 @@ impl Args for Options {
                 threshold,
             },
             OperationName::Update => Operation::Update {
-                id: id.ok_or_else(|| anyhow!("a proposal id must be provided"))?,
+                id: id.ok_or_else(|| anyhow!("a proposal-id must be provided"))?,
                 rev,
                 title,
                 description,
                 delegates,
                 threshold,
             },
             OperationName::Rebase => Operation::Rebase {
-                id: id.ok_or_else(|| anyhow!("a proposal id must be provided"))?,
+                id: id.ok_or_else(|| anyhow!("a proposal-id must be provided"))?,
                 rev,
             },
             OperationName::Show => Operation::Show {
-                id: id.ok_or_else(|| anyhow!("a proposal id must be provided"))?,
+                id: id.ok_or_else(|| anyhow!("a proposal-id must be provided"))?,
                 rev,
                 show_revisions,
             },
             OperationName::List => Operation::List,
             OperationName::Commit => Operation::Commit {
-                id: id.ok_or_else(|| anyhow!("a proposal id must be provided"))?,
+                id: id.ok_or_else(|| anyhow!("a proposal-id must be provided"))?,
                 rev,
             },
             OperationName::Close => Operation::Close {
-                id: id.ok_or_else(|| anyhow!("a proposal id must be provided"))?,
+                id: id.ok_or_else(|| anyhow!("a proposal-id must be provided"))?,
```

Comment:
I'd change all these to `a proposal must be provided`.

Reply:
https://github.com/radicle-dev/heartwood/pull/445/commits/6d0b338c6920de263d264d7bac5e5c819bc80511#diff-e46451a02ff77e76adfc85b5fb6755fe7ab32edc785b148f34005951b2cb18b0R189